test/e2e/run_e2e.sh

```diff
@@ -174,12 +174,19 @@ if [ "$e2e" = true ]; then
     #   https://github.com/kubernetes/kubernetes/blob/v1.19.2/test/e2e/network/service.go#L2522
     # * "[Fail] [sig-network] Services [It] should be able to switch session affinity for NodePort service [LinuxOnly] [Conformance]"
     #   https://github.com/kubernetes/kubernetes/blob/v1.19.2/test/e2e/network/service.go#L2538
+    #
+    # These are disabled because the nodePorts are not reachable via
+    # public IP, as the ones above and because it tries to schedule a
+    # pod directly in the node, what the admission controller only
+    # allows to system pods.
```

Comment:
I was actually wrong here. It uses the [internal IP](https://github.com/kubernetes/kubernetes/blob/v1.21.4/test/e2e/network/hostport.go#L78) and tries to connect from another pod. The only problem is that our admissions controller doesn't allow the scaling into a node:

```
err:admission webhook "pod-admitter.teapot.zalan.do" denied the request: only system pods are allowed to schedule directly on nodes
```
we might want to revisit this one and disable this check during the e2e. I will investigate further, comments are welcome.

Reply:
reactivated.

Reply:
Ok, now the problem is a bit bigger, will detail here and also in the internal issue. 

For using hostPort we will need to enable it in our [CNI configmap](https://github.com/zalando-incubator/kubernetes-on-aws/blob/51a5fc424f2457231558ad0982673d2a86fe5a18/cluster/manifests/flannel/configmap.yaml#L9-L22) like the [doc](https://v1-21.docs.kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#support-hostport) suggests. The doc also suggests we shouldn't use hostPort at all. Currently, we use it in the skipper pods but with a host network. It's a very specific use case and I don't see why to enable it for builders. Therefore, let's not enable it for now and discuss later if we want that and reactivate this test.